app/admin/custom-forms/edit/template.hbs

```diff
@@ -61,8 +61,8 @@
         <h3 class="panel-title">{{formName}}</h3>
       </div>
       <div class="panel-body">
-        {{#em-form model=previewModel submitButton=false}}
-          {{custom-form model=previewModel form=model}}
+        {{#em-form model=previewModel submitButton=false as |form|}}
+          {{custom-form model=previewModel form=model rapidForm=form}}
```

Comment:
Here we use the `form` ourselves, so I pass rapid form as `rapidForm`. Maybe it makes more sense to do `rapidForm=form` everywhere. Opinions?